### ISSUE TITLE
An application with Login, Signup  and some Pages using React framework.

### ISSUE THREAD

**Issue by SEA-Tech** [SEA-Tech/SEA-Hacktober22#3]

**Comment by DigantaChauduri06** [comment]
I can work on this project please assign me.

**Comment by JayanthkumarAllamsetty** [comment]
> I can work on this project please assign me.

Please send a pull request. So that We could assign you.

**Comment by JayanthkumarAllamsetty** [comment]
> Please send a pull request. So that We could assign you.

Please send a pull request. So that We could assign you.

**Comment by SEA-Tech** [comment]
> I can work on this project please assign me.

Yeah this is Assigned to you. You can work on it @DigantaChauduri06
